tests/DetectsChangesTest.php

```diff
@@ -103,6 +104,47 @@ public function it_can_store_the_changes_when_updating_a_model()
     }
 
     /** @test */
+    public function it_can_store_the_changes_when_updating_a_related_model()
+    {
+        $articleClass = new class() extends Article {
+            static $logAttributes = ['name', 'text', 'user.name'];
+
+            use LogsActivity;
+        };
+
+        $aUser = User::create([
+            'name' => 'a name',
+        ]);
+
+        $anotherUser = User::create([
+            'name' => 'another name',
+        ]);
+
+        $article = $articleClass::create([
+            'name' => 'name',
+            'text' => 'text',
+            'user_id' => $aUser->id,
+        ]);
+
+        $article->User()->associate($anotherUser)->save();
```

Comment:
Strange, the Article test model explicitly defines `User` as the relationship method. Lowercasing this and the model doesn't work, lowercasing this only does work. Oh well... 👍